### PR TITLE
How to Contribute section

### PR DESCRIPTION
**Pull request by SafeEHA**

### CODE REVIEW

File: contribute.html [hunk 1]
```diff
@@ -14,7 +16,18 @@ <h2 class="mk-heading mk-heading--xl mk-m-border">About Metal<sup>3</sup> Commun
...
+<header class="mk-main__header">
+<h2 class="mk-heading mk-heading--xl mk-m-border">How to Contribute</h2>
+</header>
+<ul class="mk-social-list">
+<li>Familiarize yourself with the <a href="https://github.com/orgs/metal3-io/repositories">Metal<sup>3</sup> repositories</a> on GitHub.</li>
+<li>Follow the documentation to set up your <a href="/try-it.html">development environment</a>. This will allow you to test your changes locally before submitting them.</li>
+<li>Review the existing issues, feature requests, and ongoing discussions to find areas where you can contribute</li>
+<li>Submit pull requests to improve features, fix bugs, or propose enhancements.</li>
+<li>Join <a href="https://zoom.us/j/97255696401?pwd=ZlJMckNFLzdxMDNZN2xvTW5oa2lCZz09">community calls</a> and <a href="">mailing list</a> to engage in discussion with members such as planning and design discussions.</li>
+<li>Join (P)TG meetups to help with a deeper understanding of the project as well as the future discussion.</li>
```
Comment (hroyrh) [inline contribute.html:29]: Let's move the meetup part to the community page, and just mention here to check out the community page for community resources.

File: contribute.html [hunk 2]
```diff
@@ -14,7 +16,18 @@ <h2 class="mk-heading mk-heading--xl mk-m-border">About Metal<sup>3</sup> Commun
...
-    
-</main>
+  <br>  
+<header class="mk-main__header">
+<h2 class="mk-heading mk-heading--xl mk-m-border">How to Contribute</h2>
+</header>
+<ul class="mk-social-list">
+<li>Familiarize yourself with the <a href="https://github.com/orgs/metal3-io/repositories">Metal<sup>3</sup> repositories</a> on GitHub.</li>
+<li>Follow the documentation to set up your <a href="/try-it.html">development environment</a>. This will allow you to test your changes locally before submitting them.</li>
+<li>Review the existing issues, feature requests, and ongoing discussions to find areas where you can contribute</li>
```
Comment (hroyrh) [inline contribute.html:26]: Let's just keep existing issues and feature requests here, and provide the link that lists all issues of the project.

File: contribute.html [hunk 3]
```diff
@@ -14,7 +16,18 @@ <h2 class="mk-heading mk-heading--xl mk-m-border">About Metal<sup>3</sup> Commun
...
+<li>Submit pull requests to improve features, fix bugs, or propose enhancements.</li>
```
Comment (hroyrh) [inline contribute.html:27]: This can be merged with the above line.

File: contribute.html [hunk 4]
```diff
@@ -14,7 +16,18 @@ <h2 class="mk-heading mk-heading--xl mk-m-border">About Metal<sup>3</sup> Commun
...
+<li>Join <a href="https://zoom.us/j/97255696401?pwd=ZlJMckNFLzdxMDNZN2xvTW5oa2lCZz09">community calls</a> and <a href="">mailing list</a> to engage in discussion with members such as planning and design discussions.</li>
+<li>Join (P)TG meetups to help with a deeper understanding of the project as well as the future discussion.</li>
+<li>Ask questions and seek clarifications when needed.</li>
```
Comment (hroyrh) [inline contribute.html:30]: Can be merged with "Join community calls" line.

File: contribute.html (hunk 2 above)
Comment (SafeEHA) [reply contribute.html:26]: Do we have links for feature requests?

File: contribute.html [hunk 5]
```diff
@@ -14,7 +16,28 @@ <h2 class="mk-heading mk-heading--xl mk-m-border">About Metal<sup>3</sup> Commun
...
-    
-</main>
+  <br>  
+<header class="mk-main__header">
+<h2 class="mk-heading mk-heading--xl mk-m-border">How to Contribute</h2>
+</header>
+<ul class="mk-social-list">
+<li>Familiarize yourself with the <a href="https://github.com/orgs/metal3-io/repositories">Metal<sup>3</sup> repositories</a> on GitHub.</li>
+<li>Follow the documentation to set up your <a href="/try-it.html">development environment</a>. This will allow you to test your changes locally before submitting them.</li>
+<li>Join <a href="https://zoom.us/j/97255696401?pwd=ZlJMckNFLzdxMDNZN2xvTW5oa2lCZz09">community calls</a> and <a href="https://groups.google.com/g/metal3-dev">mailing list</a> to engage in discussion with members such as planning and design discussions.</li>
```
Comment (elfosardo) [inline contribute.html:26]: let's change this to "to engage in discussions with the members of the metal3 community"

File: contribute.html [hunk 6]
```diff
@@ -14,7 +16,28 @@ <h2 class="mk-heading mk-heading--xl mk-m-border">About Metal<sup>3</sup> Commun
...
+<li>Join <a href="https://docs.openstack.org/ironic/latest/contributor/contributing.html">Ironic Community</a> for steps on how to contribute</li>
+<li>Check out <a href="/community-resources.html">community resources</a> page to find out more</li><br>
+<header class="mk-main__header">
+  <h2 class="mk-heading mk-heading--lg mk-m-border">Best Practices</h2>
+  </header>
+  <li>Review existing issues, feature requests, and ongoing discussions to find areas where you can contribute</li>
+  <li>Introduce yourself and be patient while waiting for your PR to be reviewed</li>
+  <li>If its your first time contributing, look for issues with <em>good first issue</em> labels</li>
```
Comment (elfosardo) [inline contribute.html:34]: nit: it's

**Review by elfosardo — CHANGES_REQUESTED** [review]

File: contribute.html (hunk 5 above)
Comment (SafeEHA) [reply contribute.html:26]: Line 26?

**Pull request merged**